buildpipeline/DotNet-Trusted-Publish.json

```diff
@@ -179,6 +179,7 @@
       "alwaysRun": false,
       "displayName": "packages -> dotnet.myget.org",
       "timeoutInMinutes": 0,
+      "condition": "and(succeeded(), contains(variables.PB_PublishType, 'myget'), eq(ConfigurationGroup, 'Release'))",
```

Comment:
You need to at least also condition `Update versions repository`. Honestly I would go ahead and condition the blob publishing as well given it is trivial to add, but you can do that next week if you like.